lib/view_component/translatable.rb

```diff
@@ -96,5 +96,13 @@ def translate(key = nil, **options)
     def i18n_scope
       self.class.i18n_scope
     end
+
+    def html_safe_translation(translation)
+      if translation.respond_to?(:map)
+        translation.map { |element| html_safe_translation(element) }
+      else
+        translation.html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
@foca I think removing the conditional is a reasonable conclusion, so long as it's effectively documented. Maybe a comment to this effect will help:

```suggestion
        # It's assumed here that objects loaded by the i18n backend will respond to `#html_safe?`.
        # It's reasonable that if we're in Rails, `active_support/core_ext/string/output_safety.rb`
        # will provide this to `Object`.
        translation.html_safe # rubocop:disable Rails/OutputSafety
```

Reply:
Agreed that it's a good idea to document it. Thanks.